source/modules/collection-api/account/actions/create.md.erb

```diff
@@ -1,28 +1,24 @@
 ### Account Creation
-
-```php
-<?php 
-
-use Inspetor; 
-
-$inspetor = $this->getConfiguredInspetor(); 
-
-$inspetor->trackAccountCreation($account); 
-
-?>
+```sh
+curl -XPOST https://collection-prod.inspcdn.net/account \
+    -H 'Authorization: Bearer <your API key>'
+    -d '{
+        "id":            "12345",
+        "email":         "my_user@inspetor.com",
+        "document":      "123.456.789-10",
```

Comment:
We should have no dashes and dots here

Reply:
Hm a little concerned about this... my "document" has letters in it, and what do we do for foreigners without CPFs? Should there be an additional `document` field?

Reply:
(basically the same runtime as validating that there are no dots and dashes and a lot more user friendly)

Reply:
I don't know who we would do for foreigns since if we try to use your "document" in a CPF external verification we will get an error. Anyway, don't know if we should change our api only for foreigns because how would they do the `holder_cpf`?

Reply:
> what do we do for foreigners without CPFs?

What does Ingresse do in these cases?